shub/image/push.py

```diff
@@ -39,12 +41,16 @@
 @click.option("--apikey", help="SH apikey to use built-in registry")
 @click.option("--insecure", is_flag=True, help="use insecure registry")
 @click.option("-S", "--skip-tests", help="skip testing image", is_flag=True)
+@click.option("-R", "--reauth", is_flag=True,
```

Comment:
Why `R` and not `r`?

I’m used to uppercase being used only when lowercase is already used by a different option, which does not seems to be the case. Although I see `S` is in the same situation.

Reply:
@Gallaecio  Sorry, I've just realized I missed this one. 

In a general case, lowercase is preferred indeed. However, in a case of internal options which are expected to be used pretty rarely (and can be deprecated and dropped later if the client would work fine w/o it) I'd stay with uppercase, in this way the lowercase alternatives could be used for something more important and viable. Does it make sense? If not really and it's not a matter of preference, I could rename it, no problem.

Reply:
Thanks for the explanation, it makes total sense!